src/ExpertExtender/ExpertExtender.CalendarHint.js

```diff
@@ -110,9 +113,9 @@
 
 			this.$calendarhint.children( '.' + this._prefix + '-message' ).text( msg );
 
-			this._otherCalendar = ( value.calendar() === Time.CALENDAR.GREGORIAN )
-				? Time.CALENDAR.JULIAN
-				: Time.CALENDAR.GREGORIAN;
+			this._otherCalendar = calendarModel === TimeValue.CALENDARS.GREGORIAN.uri
+				? TimeValue.CALENDARS.JULIAN.uri
+				: TimeValue.CALENDARS.GREGORIAN.uri;
 
 			msg = this._messageProvider.getMessage(
 				this._prefix + '-switch-' + this._otherCalendar.toLowerCase()
```

Comment:
This calls toLowerCase on an object or, with the simplified CALENDARS structure, on a string URI.